_extras/extra_exercises.md

```diff
@@ -22,7 +22,7 @@ or not (yet) added to the main lesson.
 > Compare it to:
 >
 > ~~~
-> left, right = [right, left]
+> left, right = right, left
```

Comment:
The solution mentions a `list`, so if we change this, we would need to change that. However, we don't teach tuples in the lesson, so we would need to take that into consideration if we make a change here that uses a `tuple`.

Reply:
Tuples are mentioned in episode **10 Defensive Programming** before the `normalize_rectangle` function. 

In the extras they are discussed some more in the **Image Grids** of the **Extras/Discussion**.

Reply:
Ah thanks for looking into this @svaberg--I tend to skip teaching the `normalize_rectangle` function part because I find the content a bit confusing, and teaching the `assert` stuff is questionable (#622, #773, #874 ). Looks like we could remove the brackets and mention `tuple` instead of `list` and link to the `tuple` reference again.

Reply:
Okay I'll get into that!

Reply:
I made this change, see below.